liana/plotting/_dotplot.py

```diff
@@ -11,6 +11,7 @@
 def dotplot(adata: anndata.AnnData = None,
             uns_key = 'liana_res',
             liana_res: pandas.DataFrame = None,
+            met: bool = False,
```

Comment:
this should go away when the names are made consistent

Reply:
depends a bit. for the metabolites we have for example ids and names, that both are useful in comparison to proteins that have only the symbol